.github/workflows/publish-command.yml

```diff
@@ -0,0 +1,64 @@
+name: integration-test
```

Comment:
is this supposed to be named integration test? or is supposed to publish?

Reply:
was misnamed here, fixed in https://github.com/airbytehq/airbyte/commit/c5dee4cc2e2e26d6b9fb0f2d1c47c74543754c4d